app/models/spree/order_decorator.rb

```diff
@@ -218,10 +218,6 @@ def update_distribution_charge!
       line_items.each do |line_item|
         if provided_by_order_cycle? line_item
```

Comment:
I'm wondering if this method can go as well. Without product distributions, all items of an order should be in the order cycle of that order, right?

What else could happen? A product can be removed from an order cycle after it has been ordered. Would that cause any problem? Could it be that they disappear from a report?

Reply:
yes, I think that is one case in which this is protecting against. Products removed from OCs.

I am not sure how the system behaves in terms of modifying orders with products that are no longer in the order cycle, or even modifying orders where the OC is already closed...
I'll add that to the test scenarios of this PR.